src/DeleteMe/DeleteMe.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
deleteme 🙄

Reply:
Yeah - sorry, that is temporary. I'm sure theres better ways to do this but needed to sync across to a Linux PC with a logic analyser attached, but remote debug from my Windows laptop running VS - will get rid of that shortly!